cypress/integration/my-orcid/my-orcid-affiliations.e2e.js

```diff
@@ -0,0 +1,53 @@
+/// <reference types="cypress" />
+
+describe.only('Affiliations' + runInfo(), () => {
+  describe.only('Employment' + runInfo(), () => {
+    it('show an option to add more items', () => {})
+    it('show an option to edit items', () => {})
+    it('display a user with no items', () => {
+      // TODO ??
+    })
+
+    describe('display a user with an item added by the same user', () => {
+      it('show the number of works as 1', () => {})
+      it('display the item privacy', () => {})
+      it('show the item details', () => {})
+      it('opens external URLs on a separate tab', () => {})
+      it('show the item edit button', () => {})
+      it('can be deleted', () => {})
+    })
+
+    describe('display a user with an item added with the API', () => {
+      it('show the number of items on the title as 1', () => {})
+      it('show the item source', () => {})
+      it('show the item "make a copy and edit" button', () => {})
+      it('can be deleted', () => {})
+    })
+
+    describe('display a user with an item added with multiple sources', () => {
+      it('show the number of items on the title as 1', () => {})
+      it('show the item multiple sources', () => {})
+      it('expand and show the multiple sources of the same item', () => {
+        // TODO ??
```

Comment:
@dandineen At the moment there are not designed to expand affiliations sources. 
I am planing to use and implement [like the one display for works](zpl.io/a3LAkD1)

But removing the checkboxes 
![image](https://user-images.githubusercontent.com/2119626/109048485-bcd54000-769c-11eb-8343-d4ee5a6fa901.png)

Reply:
@leomendoza123 That's exactly what I was thinking should happen :-) The overall behaviour should be identical to works but you can't select multiple items.